test/utils/setup.ts

```diff
@@ -132,46 +142,30 @@ export const getSafeTemplate = async (saltNumber: string = getRandomIntAsString(
     return Safe.attach(template) as Safe | SafeL2;
 };
 
-export const getSafeWithOwners = async (
-    owners: string[],
-    threshold: number = owners.length,
-    to: string = AddressZero,
-    data: string = "0x",
-    fallbackHandler: string = AddressZero,
-    logGasUsage: boolean = false,
-    saltNumber: string = getRandomIntAsString(),
-) => {
-    const template = await getSafeTemplate(saltNumber);
+export const getSafeWithOwners = async (safe: safeWithFullConfig) => {
+    if (typeof safe.threshold === "undefined") safe.threshold = safe.owners.length;
```

Comment:
You can specify default properties when descructuring: https://stackoverflow.com/questions/60582115/destructuring-properties-with-default-values-in-typescript

Reply:
Thanks! destructured 👍🏾